File: 백준/Bronze/2884. 알람 시계/알람 시계.py
Time = input().split()
H = int(Time[0])
M = int(Time[1])
if M < 45:
    M += 15
    if H == 0:
        H = 23
    else:
        H-=1
    print(H, M)
elif M >= 45:
    M -= 45
    print (H, M)

File: 백준/Bronze/2884. 알람 시계/알람 시계.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split(' ').map(Number);
const [h,m] = input

const hour = h===0?(m-45 >= 0? 0:23):(m-45 >= 0? h:h-1)
const minuite = m-45 >= 0 ? m-45: 60+(m-45)

console.log(hour+" "+minuite);